indico/modules/events/papers/forms.py

```diff
@@ -49,3 +54,38 @@ def __init__(self, *args, **kwargs):
             del self.content_reviewers
         if not self.event.cfp.layout_reviewing_enabled:
             del self.layout_reviewers
+
+
+class BulkPaperJudgmentForm(IndicoForm):
+    judgment = HiddenEnumField(enum=PaperAction)
+    contribution_id = HiddenFieldList()
+    submitted = HiddenField()
+    judgment_comment = TextAreaField(_("Comment"), render_kw={'placeholder': _("Leave a comment for the submitter...")})
+    send_notifications = BooleanField(_("Send notifications to submitter"), default=True)
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop('event')
+        super(BulkPaperJudgmentForm, self).__init__(*args, **kwargs)
+        if kwargs['judgment']:
+            self._remove_unused_fields(kwargs['judgment'])
+
+    def _remove_unused_fields(self, judgment):
```

Comment:
This can go away if we don't have any fields that are specific to certain judgment actions.